hnn_core/tests/conftest.py

```diff
@@ -63,3 +68,42 @@ def pytest_runtest_setup(item):
             # and test name
             if test_name is not None:
                 pytest.xfail("previous test failed ({})".format(test_name))
+
+
+@pytest.fixture(scope='module')
+def run_hnn_core():
+    def _run_hnn_core(backend=None, n_procs=None, n_jobs=1, reduced=False,
+                      record_vsoma=False, record_isoma=False):
```

Comment:
@kohl-carmen if you add a `postproc` parameter here, then you can avoid duplication of code